flake8_quotes/__init__.py

```diff
@@ -77,15 +77,19 @@ def get_quotes_errors(self, file_contents):
                 # ignore non strings
                 continue
 
-            if not token.string.startswith(self.inline_quotes['bad_single']):
+            # Remove any prefixes in strings like u"foobar". Use the quote
+            # character at the end of the string to know where the prefixes end
+            text = token.string[token.string.index(token.string[-1]):]
```

Comment:
There is a lot going on in this line. Can we break it down into other variables to make it easier to follow?